source/fundamentals/builders/indexes.txt

```diff
@@ -11,3 +11,181 @@ Indexes Builders
    :class: singlecol
 
 .. _indexes-builders:
+
+Overview 
+--------
+
+This guide shows you how to specify **indexes** using the Java Driver
+:doc:`builders </fundamentals/builders/>`. Index builders provide
+helper methods for the following types of indexes:
+
+- :ref:`Ascending <index-ascending>`
+- :ref:`Descending <index-descending>`
```

Comment:
Nitpick:
Just thought it was strange that the other bulleted items include "Indexes" and these don't. Should they be similar/different?